common/src/main/java/com/wynntils/services/itemfilter/statproviders/SkillStatProvider.java

```diff
@@ -38,4 +39,10 @@ public List<Integer> getValue(WynnItem wynnItem) {
                 .map(StatActualValue::value)
                 .toList();
     }
+
+    @Override
+    public List<ItemFilterType> getFilterTypes() {
+        // Skill stats are either fixed in gear
```

Comment:
Feels like there should be an "or" after this

Reply:
I thought crafted items were "identifiable", but they are not. That was the or part

Reply:
Yeah, but the comment still reads funny. 
```suggestion
        // Skill stats are fixed in gear
```
or remove the comment completely, since it is unclear if it says anything the code does not say already.

Reply:
Right, I missed that last night. I'll just do a force push with the year change then..